### PR TITLE
[Repo, Build] Remove assets folder

### PR DESCRIPTION
**Pull request by yozlet**
Implements #16, finishing #8 (with the exception of #11, which we should deal with separately)

**Note:** This PR follows #19, so merge that in first!

### CODE REVIEW

File: config/gulp/javascript.js [hunk 1]
```diff
@@ -27,6 +27,31 @@ gulp.task('eslint', function (done) {
 
 });
 
+gulp.task('copy-jquery-javascript', function (done) {
+
+  dutil.logMessage(task, 'Copying JS from jQuery');
+
+  var stream = gulp.src('./node_modules/jquery/dist/jquery.min.*')
+    .pipe(gulp.dest('assets/js/vendor/jquery/'));
```
Comment (juliaelman) [inline config/gulp/javascript.js]: Is there a reason this needs to live inside of a folder? Or can the file live in the `vendor` folder on it's own?

File: config/gulp/javascript.js [hunk 2]
```diff
@@ -27,6 +27,31 @@ gulp.task('eslint', function (done) {
...
+
+/* TODO: Remove this build step *and* its invocation in the main task
+   once build-time syntax colouring is restored
+*/
+gulp.task('copy-prismjs-javascript', function (done) {
+
+  dutil.logMessage(task, 'Copying JS from PrismJS');
+
+  var stream = gulp.src('./node_modules/prismjs/prism.js')
+    .pipe(gulp.dest('assets/js/vendor/prismjs/'));
```
Comment (juliaelman) [inline config/gulp/javascript.js]: Is there a reason this needs to live inside of a folder? Or can the file live in the `vendor` folder on it's own?

**Comment by juliaelman** [comment]
Looks good! Added a few notes inline and let me know if those make sense or not.

Also, I submitted a PR to add the close images to the `img` folder, refs #20

**Comment by yozlet** [comment]
@juliaelman I put them in folders because it felt like a better way to organise it, but I don't have massively persuasive arguments in favour of the change, so I'm happy to leave it how it was if you prefer.

**Comment by juliaelman** [comment]
@yozlet cool! I just wasn't sure if they were being put into folders b/c of reasons 😄  Yeah, let's change that so that files output to the `vendor` directory itself - thank you!

**Comment by yozlet** [comment]
Gah, I'm an idiot. I forgot that I ripped all that vendor-lib-copying code out because Browserify doesn't need it.

**Pull request merged**